sdk/python/feast/type_map.py

```diff
@@ -566,3 +577,51 @@ def _non_empty_value(value: Any) -> bool:
     return value is not None and (
         not isinstance(value, Sized) or len(value) > 0 or isinstance(value, str)
     )
+
+
+def spark_to_feast_value_type(spark_type_as_str: str) -> ValueType:
+    # TODO not all spark types are convertible
```

Comment:
can you give some examples of types that can't be converted?

Reply:
Done.